src/main/java/org/kuali/kfs/pdp/businessobject/PaymentGroup.java

```diff
@@ -1158,19 +1159,25 @@ public void setVendorAddress(VendorAddress addr) {
      * @param string
      */
     public void setPayeeIdAndName(String string) {
-        payeeId = string;
-        
-        BusinessObjectService bos = SpringContext.getBean(BusinessObjectService.class);
-        String[] headerDetails = payeeId.split("-");      
-        Map<String, String> fieldValues = new HashMap<String, String>();
-        fieldValues.put("vendorHeaderGeneratedIdentifier", headerDetails[0]/*payeeId*/);
-        fieldValues.put("vendorDetailAssignedIdentifier", headerDetails[1]);
-        
-        List<VendorDetail> details = (List<VendorDetail>)bos.findMatching(VendorDetail.class, fieldValues);
-        if (details.size() == 1) {
-            payeeName=details.get(0).getVendorName();
+        if (string.split("-").length == 1) {
```

Comment:
We had a previously customized PaymentGroup and that customization presumed it was just vendors, and not employee IDs.  I am not sure if this solution is what we need yet.  Marcia gave me some more details at testing PDP.